executor/platform_compat/unix/pty_manager.py

```
@@ -0,0 +1,167 @@
+    ) -> UnixPtyProcess:
+        """Spawn a new process with a PTY."""
+        # Create pseudo-terminal pair
+        master_fd, slave_fd = pty.openpty()
+
+        # Set up environment
+        if env is None:
+            env = os.environ.copy()
+        env["TERM"] = env.get("TERM", "xterm-256color")
+
+        # Spawn process
+        process = subprocess.Popen(
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)